edward/inferences/inference.py

```diff
@@ -169,11 +171,25 @@ def initialize(self, n_iter=1000, n_print=None, scale=None, logdir=None,
     logdir : str, optional
       Directory where event file will be written. For details,
       see ``tf.summary.FileWriter``. Default is to write nothing.
+    logrun : str, optional
```

Comment:
They way I thought about it is that `logrun` will tend to be pretty static for a given project and you want a way to cleanly separate info each time you refit your models.  Therefor you can leave it blank as you arbitrarily refine the model and use the timestamps to track your progress.  Else you could set `logrun` to something like `using_adam` vs `using_adagrad` to highlight in TensorBoard the convergence results of each.

I wanted to keep `logdir` a string since it is a string in TensorBoard, and given it has the same name I wanted to make the usage similar to avoid confusion.